dataStorage/storage.js

```diff
@@ -31,14 +31,14 @@ async function loadAsync(model, id, update) {
     let data = await loadFromOffilneStorageAsync(model.key, id);
     if (!data || data.length == 0) {
         // TODO need to sync from server;
-        data = await loadFromCloudAsync(model, id);
+        data = await loadFromCloudAsync(model, id, false);
         data && saveToOffilneStorageAsync(data, model.key, id);
     } else if (update) {
         // update the offline storage silently
-        loadFromCloudAsync(model, id)
-        .then((cloudData) => {
-            saveToOffilneStorageAsync(cloudData, model.key, id);
-        });
+        loadFromCloudAsync(model, id, true)
+            .then((cloudData) => {
```

Comment:
Why use ".then" here, but "await" in other places? Would it be better to be consistent?